draft-ietf-quic-http.md

```diff
@@ -496,13 +496,9 @@ The PRIORITY frame payload has the following fields:
   : A two-bit field indicating the type of element being depended on.
 
   Empty:
-  : A three-bit field which MUST be zero when sent and MUST be ignored
+  : A four-bit field which MUST be zero when sent and MUST be ignored
     on receipt.
```

Comment:
Not really for this, but do we have to mandate ignoring reserved fields?

Reply:
If you don't, you risk an overzealous implementation enforcing the "MUST be zero" and we can't use it later without negotiation.  "MUST ignore" seems cleaner than "MUST not die in flames if it's not zero."